test/commands/skuid/page/pull.test.ts

```diff
@@ -12,6 +12,17 @@ const v1PageObject = {
   "maxAutoSaves": 100,
   "body": "<skuidpage><models/></skuidpage>",
 };
+
+const v1PageObjectWithSlashModule = {
+  "apiVersion": "v1",
+  "name": "SomePageName",
```

Comment:
let's also include slashes in the page name to ensure that the code replaces slashes in name as well as module, since you can include slashes in your page name as well

Reply:
Changed "SomePageName" to "Some/Page\\Name"